pkg/registry/apis/dashboard/search.go

```diff
@@ -63,7 +63,7 @@ func (s *SearchHandler) GetAPIRoutes(defs map[string]common.OpenAPIDefinition) *
 				Spec: &spec3.PathProps{
 					Get: &spec3.Operation{
 						OperationProps: spec3.OperationProps{
-							Tags:        []string{"Search"},
+							Tags:        []string{"Search", "Folder", "Dashboard"},
```

Comment:
Basically the knock-on effect for this is that the frontend API client needs to know that this endpoint "provides" entities of types `search results`, `folders`, and `dashboards`. This is so that other parts of the UI code can know to invalidate any caches it has of these types - e.g. we need the UI to know that if a folder is created or deleted, then it will need to refetch any results provided by the search endpoint